paddle/majel/allocation.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <memory>
+
+#include <majel/place.h>
+#include "hl_gpu.h"
```

Comment:
The ```hl_gpu.h``` file path has already been set into the cmake projects. I will implement a simple malloc library for majel instead of using paddle/cuda